content/blog/2017-hashcode.md

```diff
@@ -0,0 +1,50 @@
+---
+date: "2017-02-26T00:00:00Z"
+title: CPSSD Tackles Hashcode 2017
+---
+
+Google's [Hashcode][1] qualification round took
+place on the 23rd of February, and lots of CPSSD first and second years crowded
+in the DCU Hashcode hub to compete.
+
+<!--more-->
```

Comment:
Does this comment do something?

Reply:
Yes. It allows you to define the end of a summary. So if I have a long article, and I  place `<!--more-->` after the first paragraph, the article list would show only the first paragraph, and you will have to click on the article to view the whole thing.